website/docs/cloud-docs/api-docs/stacks.mdx

```diff
@@ -0,0 +1,2724 @@
+| `GET` | `/stack-configurations/:stack_configuration_id/stack-diagnostics` | Call this endpoint to [list stack configuration diagnostics](#list-stack-configuration-diagnostics).|
+| `GET` | `/stacks/:stack_id/stack-plans` | Call this endpoint to [list stack plans](#list-stack-plans).|
+| `GET` | `/stack-plans/:stack_plan_id/` | Call this endpoint to [show stack plan](#show-stack-plan).|
+| `GET` | `/stack-plans/:stack_plan_id/stack-events` | Call this endpoint to [list stack events](#list-stack-events).|
+| `GET` | `/stack-events/:stack_event_id` | Call this endpoint to [show stack event](#show-stack-event).|
+| `GET` | `/stack-plans/:stack_plan_id/plan-description` | Call this endpoint to [show plan description](#show-plan-description).|
+| `GET` | `/stack-plans/:stack_plan_id/apply-description` | Call this endpoint to [show apply description](#show-apply-description).|
+| `GET` | `/stack-configurations/:stack_configuration_id/stack-plans` | Call this endpoint to [list stack configuration plans](#list-stack-configuration-plans).|
+| `GET` | `/stacks/:stack_id/stack-states` | Call this endpoint to [list stack states](#list-stack-states).|
+| `GET` | `/stack-states/:stack_state_id` | Call this endpoint to [show stack states](#show-stack-state).|
+| `GET` | `/stack-states/:stack_state_id/description` | Call this endpoint to [show stack state description](#show-stack-state-description).|
+| `POST` | `/stacks` | Call this endpoint to [create a stack](#create-a-stack).|
+| `POST` | `/stack-diagnostics/:stack_diagnostic_id/acknowledge` | Call this endpoint to [acknowledge stack diagnostic](#acknowledge-stack-diagnostic).|
+| `POST` | `/stacks/:stack_id/stack-deployments/:deployment_name/create-destroy-plan` | Call this endpoint to [create destroy plan](#create-destroy-plan).|
+| `POST` | `/stacks/:stack_id/stack-deployments/:deployment_name/delete-deployment` | Call this endpoint to [delete deployment](#delete-deployment).|
+| `POST` | `/stacks/:stack_id/actions/update-configuration` | Call this endpoint to [fetch a stack configuration](#fetch-the-latest-stack-configuration).|
+| `POST` | `/stacks/:stack_id/delete` | Call this endpoint to [delete a stack](#safe-delete-stack).|
+| `POST` | `/stacks/:stack_id/force-delete` | Call this endpoint to [force delete a stack](#force-delete-stack).|
+| `POST` | `/stack-plans/:stack_plan_id/approve` | Call this endpoint to [approve a stack plan](#approve-stack-plan).|
+| `POST` | `/stack-plans/:stack_plan_id/discard` | Call this endpoint to [discard a stack plan](#discard-stack-plan).|
+| `POST` | `/stack-plans/:stack_plan_id/cancel` | Call this endpoint to [cancel a stack plan](#cancel-stack-plan).|
+| `POST` | `/stack-plans/:stack_plan_id/replan` | Call this endpoint to [replan stack plan](#replan-stack-plan).|
+| `PATCH` | `/stacks/:stack_id` | Call this endpoint to [update a stack](#update-a-stack).|
+
+## List stacks
```

Comment:
I wonder if it would be helpful to have a "Requirements" section above this where we can talk about specific permissions/tokens are user requires to interact with this endpoints? We did something similar on the Projects API page, see https://developer.hashicorp.com/terraform/cloud-docs/api-docs/projects#requirements.